content/docs/hooks-intro.md

```diff
@@ -25,86 +25,86 @@ function Example() {
 }
 ```
 
-This new function `useState` is the first "Hook" we'll learn about, but this example is just a teaser. Don't worry if it doesn't make sense yet!
+Funkcja `useState` jest pierwszym "Hookiem", o którym będziemy się uczyć. Przykład ten jest jednak zaledwie zwiastunem. Nie przejmuj się, jeżeli nie ma to jeszcze większego sensu!
```

Comment:
```suggestion
Funkcja `useState` jest pierwszym "hookiem", o którym będziemy się uczyć. Przykład ten jest jednak zaledwie zwiastunem. Nie przejmuj się, jeżeli nie ma to jeszcze większego sensu!
```

Reply:
Moim zdaniem lepiej zaczynać "hooki" małą literą.

Reply:
Chciałem być wierny oryginałowi w tej kwestii - z jakiegoś powodu wszędzie użyli wielkiej litery.

Reply:
Może podeszli do tego jak do hasła reklamowego, nazwy własnej ;-) W języku polskim nadal, imho, lepiej pasuje pisać małą literą.

Reply:
Sprawdziłem Hiszpański, też jest wielką literą:
https://es.reactjs.org/docs/hooks-intro.html
Może skonsultować się z twórcami?

Reply:
[Skonsultowałem się z "twórcami"](https://twitter.com/dan_abramov/status/1100933020536721408) :P i padła odpowiedź, że możemy pisać jak chcemy. Oni piszą wielką literą, bo chcieli nadać tej nazwie pewien stopień "specjalności".

Reply:
Dla mnie dlatego pisanie od małej litery jest lepsze, bo to nie jest do końca ich wymyślona, własna nazwa. [Hooki istnieją w programowaniu od lat](https://pl.wikipedia.org/wiki/Hook_(informatyka)) i nie są niczym wyjątkowym - ot, kolejny mechanizm. Gdyby iść za ich schematem, to trzeba by pisać wielką literą "Props", "Lifecycle", "Components" - a tego tak nie piszemy :-)

Reply:
Ok, masz rację - zmienię na małą. Jak ktoś jest przeciwko, teraz jest dobry czas na sprzeciw ;).

Reply:
@cvqprs @iamarek @TeoTN @bartlomiejzuber @marcinostrouch @jakubdrozdek - może głosowanie? 🚀 - wielką, 👀 - małą